### PR TITLE
editor for edit form, fix format homepage, strip_tags

### PR DESCRIPTION
**Pull request by phuongha47**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## WHAT this PR does?
- Editor for edit form, fix format homepage, strip_tags

## Notes Impacted Areas:
- N/A.

## Attachments:
- 
image

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -21,11 +13,21 @@
...
-    return view('welcome');
-});
+Route::get('/', 'App\Http\Controllers\Users\HomeController@show')->name('home.index');
+Route::get('/show/{id}', 'App\Http\Controllers\Users\HomeController@show')->name('home.category');
+Route::get('/search', 'App\Http\Controllers\Users\HomeController@search')->name('home.search');
+
+Route::get('/category/show/{id}', 'App\Http\Controllers\Users\UserCategoryController@showSubCategories')
+    ->name('userCategory.show');
+Route::get('/category/showCategory/{id}', 'App\Http\Controllers\Users\UserCategoryController@showCategories')
+    ->name('userCategory.showCategory');
```
Comment (Khoacannotcode) [inline routes/web.php:27]: Có route mới rồi đó, chỗ này chắc conflict

Comment (phuongha47) [reply routes/web.php:27]: okie

**Comment by Khoacannotcode** [comment]
ready

**Comment by phuongha47** [comment]
ready

**Pull request merged**